### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by kakkoyun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+[![Build Status](https://circleci.com/gh/observatorium/operator.svg?style=svg)](https://circleci.com/gh/observatorium/operator)
+
+Operator deploying the Observatorium project.
+Currently, this includes:
+
+0. operator itself.
+0. example configuration for deploying Observatorium via the Observatorium Operator.
+
+For more information on the operator, see the [Observatorium Operator documentation](./doc/operator/deploy-operator.md).
```
Comment (rollandf) [inline README.md:11]: The directoy is `docs`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Observatorium Operator
+
+[![Build Status](https://circleci.com/gh/observatorium/operator.svg?style=svg)](https://circleci.com/gh/observatorium/operator)
+
+Operator deploying the Observatorium project.
+Currently, this includes:
+
+0. operator itself.
```
Comment (squat) [inline README.md:8]: I don't have a strong opinion here, but this makes the list zero-indexed, should we make it start with a 1?

Comment (squat) [inline README.md:8]:
```suggestion
1. the operator itself; and
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+0. example configuration for deploying Observatorium via the Observatorium Operator.
```
Comment (squat) [inline README.md:9]:
```suggestion
1. example configuration for deploying Observatorium via the Observatorium Operator.
```

File: README.md (hunk 2 above)
Comment (kakkoyun) [reply README.md:8]: GH markdown handles it if I'm not mistaken.

**Comment by metalmatze** [comment]
Thanks for helping with this. I'm kind of waiting for the PR in the deployments repo to merge so that I can update everything accordingly.

**Comment by metalmatze** [comment]
Oh, seems like we already merged it :tada:

File: README.md (hunk 2 above)
Comment (squat) [reply README.md]: not exactly, this is the result of
```
0. x
1. y
2. z
```

0. x
1. y
2. z

See that it becomes zero-indexed

**Review by squat — APPROVED** [review]

**Pull request merged**